tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -0,0 +1,358 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<rawdata_files>
+
+  <!-- ======================== -->
+  <!-- lai -->
+  <!-- ======================== -->
+
+  <mksrf_flai>
+    <entry>
+      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.histsimyr1850-2015.c20230226/mksrf_pftlaihgt_ctsm52_histLUH2_2005.c20230226.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_0.25x0.25_nomask_cdf5_c200129.nc</mesh_filename>
+    </entry>
+    <entry pft_years="2005" hires_pft='on'>
+      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.05x0.05.LUH2.histsimyr2005.c190116/mksrf_lai_histclm52deg005_earthstatmirca_2005.cdf5.c220228.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_cdf5_c200129.nc</mesh_filename>
+    </entry>
+  </mksrf_flai>
+
+  <!-- ======================== -->
+  <!-- soil properties: fmax, soil-texture (sand/clay, orgc), and soil color-->
+  <!-- ======================== -->
+
+  <!-- <mksrf_firrig> -->
```

Comment:
@mvertens this is commented out and it looks like it's something you according to git. I'm wondering the reason you commented it out? I'm worried it might be something that we need in place and we are missing it? Or is it just commented out code that isn't needed and we can removed? Thanks in advance for your advice here.

Reply:
@ekluzek - its been a really long time since I implemented the new esmf mksurfdata code and I don't remember why I commented this out or whether it might be useful in the future. I know that @slevis-lmwg and I did a very careful validation of the results - so he might be able to weigh in as to why we were not using this.

Reply:
I do not remember a discussion on irrigation during the development and evaluation process.
@ekluzek we're meeting in 15', let's address this in the same meeting if we have time.

Reply:
Handled in cf250338574522a4c4a9f138ca8192c9aa203e23 in #2378 so resolving this conversation.